debug/testlib.py

```diff
@@ -947,6 +948,33 @@ def __enter__(self):
     def __exit__(self, _type, _value, _traceback):
         self.gdb.pop_state()
 
+
+def load_excluded_tests(excluded_tests_file, target_name):
+    result = []
+    if excluded_tests_file is None or len(excluded_tests_file) == 0:
+        return result
+
+    target_excludes = {}
+    with open(excluded_tests_file) as file:
+        raw_data = yaml.safe_load(file)
+        invalid_format_message = "Invalid format for the exclusion file. Please take a look at the " \
+                                 "\"excluded.yaml.example\" file as an example of the test exclusion file format."
+        for test_list in raw_data.values():
+            if not isinstance(test_list, list):
+                raise ValueError(invalid_format_message, excluded_tests_file, test_list)
```

Comment:
Specifically here, can you tell the user something like "list of tests wasn't a list".

Even better would be to do `for (target, test_list) in raw_data.items()` above so the error can include the name of the target the list belongs to.

Reply:
Done